authors/apps/profiles/views.py

```diff
@@ -61,3 +63,64 @@ class UserProfileListView(generics.ListAPIView):
     @classmethod
     def get_queryset(self):
         return Profile.objects.all()
+
+
+class IFollowListView(generics.RetrieveAPIView):
+    """A class for following and or unfollowing other authors."""
+    permission_classes = (IsAuthenticated,)
+    serializer_class = FollowerSerializer
+
+    @classmethod
+    def get(cls, request):
+        # returns a list of actors who request.user is following
+        data = {'id': request.user.pk}
+        following = Follower.objects.values_list('author', flat=True).filter(follow=request.user.username)
+        response = {'message': f'You follow : {following}', 'status_code': status.HTTP_200_OK}
```

Comment:
@ManuelDominic, create a renderer class for the following views as done in other views